core/invoker/src/main/scala/whisk/core/container/Container.scala

```diff
@@ -59,18 +59,18 @@ class Container(
         s"container [$name] [$id] [$ip]"
     }
 
-    def pause(): Unit =
+    def pause(): Boolean =
         if (useRunc) {
-            RuncUtils.pause(containerId)
+            Container.isRuncSuccessfull(RuncUtils.pause(containerId))
         } else {
-            pauseContainer(containerId)
+            Container.isDockerSuccessful(pauseContainer(containerId))
         }
 
-    def unpause(): Unit =
+    def unpause(): Boolean =
         if (useRunc) {
-            RuncUtils.resume(containerId)
+            Container.isRuncSuccessfull(RuncUtils.resume(containerId))
         } else {
-            unpauseContainer(containerId)
+            Container.isDockerSuccessful(unpauseContainer(containerId))
```

Comment:
did you consider moving this to `DockerOutput` as in `DockerOutput.isSuccessful` and provide the same in `RuncUtils`?